### PR TITLE
Header 28

### PR DESCRIPTION
**Pull request by kenangsmv**

### CODE REVIEW

**Review by idriska — APPROVED** [review]

File: editor-components/header/header28/header28.module.scss [hunk 1]
```diff
@@ -1,288 +1,247 @@
...
+          color: rgba(255, 255, 255, 0.56);
+          line-height: 1.2;
+          margin: 0;
+        }
+
+        .tag {
+          // color: var(--composer-font-color-primary);
+          // --composer-font-color-primary this variable defined 2 times
+          // one with black, one with white, need to fix
+          color: white;
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.module.scss:112]:
```suggestion
          color: white;
```

File: editor-components/header/header28/header28.tsx [hunk 2]
```diff
@@ -1,174 +1,349 @@
...
+          type: "object",
+          key: "slide",
+          displayer: "Slide",
+          value: [
+            {
+              type: "image",
+              key: "image",
+              displayer: "Image",
+              value:
+                "https://viseo.progressionstudios.com/wp-content/uploads/2017/04/dark-poison-large-1400x700.jpg",
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.tsx:56]: must be spica storage url

File: editor-components/header/header28/header28.tsx [hunk 3]
```diff
@@ -1,174 +1,349 @@
...
+          type: "object",
+          key: "slide",
+          displayer: "Slide",
+          value: [
+            {
+              type: "image",
+              key: "image",
+              displayer: "Image",
+              value:
+                "https://viseo.progressionstudios.com/wp-content/uploads/2017/04/front-lines-1400x700.jpg",
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.tsx:100]: must be spica storage url

File: editor-components/header/header28/header28.tsx [hunk 4]
```diff
@@ -1,174 +1,349 @@
...
+          type: "object",
+          key: "slide",
+          displayer: "Slide",
+          value: [
+            {
+              type: "image",
+              key: "image",
+              displayer: "Image",
+              value:
+                "https://viseo.progressionstudios.com/wp-content/uploads/2017/04/dep-space-1400x700.jpg",
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.tsx:144]: must be spica storage url

File: editor-components/header/header28/header28.tsx [hunk 5]
```diff
@@ -1,174 +1,377 @@
...
+                  className={`${this.decorateCSS("video-player-container")} ${
+                    this.getComponentState("play-video") &&
+                    this.decorateCSS("video-player")
+                  }`}
+                  onClick={() => {
+                    this.setComponentState("play-video", false);
+                  }}
+                >
+                  {this.getComponentState("play-video") && (
+                    <iframe
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.tsx:275]: add classname

File: editor-components/header/header28/header28.tsx [hunk 6]
```diff
@@ -1,174 +1,377 @@
...
+                    ></iframe>
+                  )}
+                  {this.getPropValue("close_icon") && (
+                    <div
+                      className={this.decorateCSS("close-button")}
+                      onClick={() =>
+                        this.setComponentState("play-video", false)
+                      }
+                    >
+                      <ComposerIcon name={this.getPropValue("close_icon")} />
```
Comment (Ayazelar) [inline editor-components/header/header28/header28.tsx:292]: add classname

**Review by Ayazelar — CHANGES_REQUESTED** [review]
.

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**